back-end/lms/models/student.py

```diff
@@ -32,7 +32,56 @@ class Student(Person):
     birthdate = models.DateField()
     program = models.ForeignKey(Program, models.DO_NOTHING)
     status = models.ForeignKey(Status, models.DO_NOTHING)
-    photo = models.CharField(max_length=128, blank=True, null=True)
+    photo = models.ImageField(upload_to = '', max_length=128)
+    place_birth = models.TextField()
+    region = models.TextField()
+    adress = models.TextField()
+    pasport = models.TextField() #можно числовой, но проще так
+    pasport_ufms = models.TextField()
+    pasport_ufms_code = models.TextField()
+    pasport_date = models.DateField()
+    citizence = models.TextField()
+    fio_rod = models.TextField()
+    hse_id = models.TextField()
+    faculty = models.TextField()
+    edu_program = models.TextField()
+    hse_group = models.TextField()
```

Comment:
I would also make a separate model for these fields too. One model for the `faculty` + `edu_program`, one model for `hse_group` that has a foreign key to the previous model. You can look at the `Milgroup` and `Milfaculty` example for that.

Reply:
You can also reuse prorgrams and groups this way. This will lead to less mistakes in the future